test/conformance/tests/sriov_operator.go

```diff
@@ -899,7 +899,7 @@ var _ = Describe("operator", func() {
 				sriovDevice := unusedSriovDevices[0]
 
 				createSriovPolicy(sriovDevice.Name, testNode, 5, resourceName)
-				ipam := `{"type": "host-local","ranges": [[{"subnet": "3ffe:ffff:0:01ff::/64"}]],"dataDir": "/run/my-orchestrator/container-ipam-state"}`
+				ipam := `{"type":"host-local","subnet":"10.10.10.0/24","rangeStart":"10.10.10.171","rangeEnd":"10.10.10.181","routes":[{"dst":"0.0.0.0/0"}],"gateway":"10.10.10.1"}`
```

Comment:
why you change the subnet to use ipv4?

Reply:
This test case doesn't related to ipv6. So i decided to use ipv4 protocol instead